### DISCUSSION TITLE
Issue - ACU to conditions default=all

### DISCUSSION THREAD

**Discussion by DarkevilPT** [kwankiu/acu#13] [Q&A]
Hey Samuel ive found yesterday a loop on acu when trying to do acu -y install qt5

So basically the prompt is default:All
Requires an enter press

So I assume that to all your efforts theres also this if to be added to all others you have refreshed successfully added.

**Comment by kwankiu** [comment]
acu install qt5 --noconfirm

**Reply by DarkevilPT** [reply]
Dont u want to add automation for that --noconfirm when needed? Or its risky towards other managers etc etc?
